lib/web/apiserver.go

```diff
@@ -1379,6 +1388,33 @@ func (h *Handler) oidcCallback(w http.ResponseWriter, r *http.Request, p httprou
 	return redirectURL.String()
 }
 
+func (h *Handler) installer(w http.ResponseWriter, r *http.Request, p httprouter.Params) (interface{}, error) {
+	httplib.SetScriptHeaders(w.Header())
+	installer, err := h.auth.proxyClient.GetInstaller(r.Context())
+	if err != nil {
+		if trace.IsNotFound(err) {
+			installer = installers.DefaultInstaller
+		} else {
+			return nil, trace.Wrap(err)
+		}
```

Comment:
I don't think this fallback logic belongs in the web handler. Otherwise e.g. tctl get installer will return "not found" if you don't have the installer. This fallback should be implemented in `GetInstaller` so if you call that API method and there's not custom installer, a default object is returned.